ci-operator/step-registry/openshift-observability/enable-cluster-logging/openshift-observability-enable-cluster-logging-commands.sh

```diff
@@ -0,0 +1,605 @@
+        enable_clf_5x
+    else
+        enable_clf_6x
+    fi
+
+    echo " deploy LogFileMetricExporter "
+oc apply -f - <<EOF
+apiVersion: logging.openshift.io/v1alpha1
+kind: LogFileMetricExporter
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  nodeSelector: {}
+EOF
+}
+
+### Create application and alerts
+function create_pod_alert()
+{
+    user_namespace=$1
+    echo "# Create app/alert under namespace ${user_namespace}"
+    prepare_namespace "$1"
+
+    echo "create pod centos-logtest in ${user_namespace}"
+    cat <<EOF|oc -n $user_namespace create -f -
+---
+apiVersion: v1
+data:
+  ocp_logtest.cfg: |
+    --num-lines 0 --line-length 200 --word-length 9 --rate 60 --fixed-line
+kind: ConfigMap
+metadata:
+  name: logtest-config
+
+---
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  labels:
+    run: centos-logtest
+    test: centos-logtest
+  name: centos-logtest
+spec:
+  replicas: 1
+  template:
+    metadata:
+      generateName: centos-logtest-
+      labels:
+        run: centos-logtest
+        test: centos-logtest
+    spec:
+      containers:
+        - env: []
+          image: quay.io/openshifttest/ocp-logtest@sha256:6e2973d7d454ce412ad90e99ce584bf221866953da42858c4629873e53778606
+          imagePullPolicy: Always
+          name: centos-logtest
+          resources: {}
+          terminationMessagePath: /dev/termination-log
+          volumeMounts:
+            - mountPath: /var/lib/svt
+              name: config
+      imagePullSecrets:
+        - name: default-dockercfg-ukomu
+      volumes:
+        - configMap:
+            name: logtest-config
+          name: config
+EOF
+
+    echo "create alertingrule in ${user_namespace}"
+    cat <<EOF |oc -n ${user_namespace} apply -f -
+apiVersion: loki.grafana.com/v1
+kind: AlertingRule
+metadata:
+  labels:
+    openshift.io/log-alerting: "true"
+  name: dev-workload-alerts
+spec:
+  groups:
+    - interval: 1m
+      name: devAppAlert
+      rules:
+        - alert: DevAppLogVolumeIsHigh
+          annotations:
+            description: My application has high amount of logs.
+            summary: project "{${user_namespace}" log volume is high.
+          expr: >
+            count_over_time({kubernetes_namespace_name="${user_namespace}"}[2m])
+            > 10
+          for: 5m
+          labels:
+            severity: info
+            devApp: 'true'
+  tenantID: application
+EOF
+
+    echo "Note: you can grant the common user to access logs the alerts using below command"
+    echo " oc -n ${user_namespace} policy add-role-to-user view  <test_user_x>"
+    echo " oc -n ${user_namespace} policy add-role-to-user cluster-logging-application-view <test_user_x>"
+    echo " oc -n ${user_namespace} policy add-role-to-user monitoring-rules-edit <test_user_x>"
+    echo " oc -n ${user_namespace} policy add-role-to-user cluster-monitoring-view <test_user_x> "
+}
+
+function create_infra_aduit_alerts()
+{
+   echo "Create Infra and audit alert"
```

Comment:
Per current setting, audit logs are not collected.

Reply:
Added audit logs.